cmd/status.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// statusCmd represents the status command
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Check if the HTTP server is running",
+	Long:  `Check if the HTTP server has been started and answer 200 for /status.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		port := os.Getenv("PORT")
+		if len(port) == 0 {
+			port = "8080"
```

Comment:
We should let viper handle the whole config or ENV or default. Can be the object of a separate PR.